lib/jira/commands/comment.rb

```diff
@@ -3,15 +3,37 @@ class CLI < Thor
 
     desc "comment", "Add a comment to the input ticket"
     def comment(ticket=Jira::Core.ticket)
-      comment = self.io.ask("Leave a comment for ticket #{ticket}")
-      if comment.strip.empty?
-        puts "No comment posted."
-      else
-        comment.gsub!(/\@[a-zA-Z]+/,'[~\0]').gsub!('[~@','[~')
+      comment = self.get_comment_body(ticket)
+      if !comment.empty?
         self.api.post("issue/#{ticket}/comment", { body: comment }) do |json|
           puts "Successfully posted your comment."
+          return
         end
       end
+      puts "No comment posted."
+    end
+
+    desc "commentd", "Delete a comment to the input ticket"
+    def commentd(ticket=Jira::Core.ticket)
```

Comment:
Sure. But this is nit-picky given the fact you plan to re-architect the code, right?